### ISSUE TITLE
Caching demo - why do some movies have no timing?

### ISSUE THREAD

**Issue by TobyHFerguson** [redis-developer/redis-microservices-demo#27]
If in the caching demo I go further down the list of movies and select one (Kingsman, for example), then I see no timing for the query:

![image](https://user-images.githubusercontent.com/6525189/96629252-3791e700-12c8-11eb-85cb-258395a79003.png)

Furthermore, the little arrow icon is disabled. Why is this?

**Comment by tgrall** [comment]
Some of the movies do not have an IMBD_ID.

The last column of this SQL command is supposed to be the IMBD ID, and in fact these records are missing:

* IMDB ID
* Poster URL
* Plot

When I have started to write this application my script that generates the SQL statement from IMBD API did not have all the fields. Then I have improved it to add more data.

Since, I do not want to add manually all fields manually, I will change the UI to indicate that OMBD API cannot be called when the record does not have the proper values.
